### DISCUSSION TITLE
line on plot does not appear

### DISCUSSION THREAD

**Discussion by manuelatito** [teamalgoritma/community#880] [Q&A] [answered]
Dear Team Algoritma, 

Saya ingin bertanya, mgp line pada plot yg saya buat ini tidak muncul ya, hanya muncul titiknya saja? Apakah ada yg salah dgn code saya? Terima kasih

![image](https://github.com/teamalgoritma/community/assets/164650405/975f8d72-8dcd-49bc-990b-3d2f5e8921dc)

![image](https://github.com/teamalgoritma/community/assets/164650405/3b08e1ae-e493-49a9-9385-853e64082e98)

**Comment by ramdhinafinita** [comment]
Dear @manuelatito 
Terima kasih telah mengajukan pertanyaan melalui forum ini.
Kami akan kembali dengan jawaban kami pada hari Senin – Jumat pukul 10:00 – 18:00 WIB.
Best Regards,
Team Algoritma

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @manuelatito 
Terimakasih sudah menunggu jawaban dari pertanyaan yang diajukan. Untuk kendala disebabkan karena penempatan aes(text = label) seharusnya dimasukkan kdelaam aes di ggplot bukan di masukkan kedalam geom_point.

Bisa coba untuk sesuaikan dengan kode berikut:
```
plot_avg <- ggplot(data = plot_4, 
                mapping = aes(x =restaurant, 
                              y = avg_calories,
                              text=label))+
  geom_line(group= 1, col="red") +
  geom_point(col="black") +
  scale_y_continuous(labels = comma) +
  labs(
    title = "Rata-rata Kalori Makanan di Tiap Restaurant",
    x = "Restaurant",
    y = "Average Calories"
  ) +
  theme_minimal()
  

ggplotly(plot_avg, tooltip = "text")
```

bisa di coba terlebih dahulu semoga berhasil

**Reply by manuelatito** [reply]
Dear kak @ramdhinafinita,
Terima kasih kak Fini untuk jawabannya, sudah berhasil kak..

**Reply by ramdhinafinita** [reply]
Sama sama Ibu :)

**Reply by manuelatito** [reply]
Dear kak @ramdhinafinita,
Kak Fini, maaf kak sy ada pertanyaan lanjutan..
Interactive plot sy tdk keluar sepenuhnya setelah di knit, hasilnya seperti ini kak

![image](https://github.com/teamalgoritma/community/assets/164650405/5093f454-8de0-41c5-acb2-151abe242b41)

![image](https://github.com/teamalgoritma/community/assets/164650405/c112f4a9-4075-44b9-8001-b852057d51aa)

![image](https://github.com/teamalgoritma/community/assets/164650405/96bda546-c373-45ce-be3e-f66f21b2bab9)
 
Mohon bantuannya kak, terima kasih

**Reply by ramdhinafinita** [reply]
Dear @manuelatito,

untuk plotly atau tooltip tidak muncul bisa dicek kembali pada kode yang digunakan apakah parameter text sudah dimuat pada ggplotnya. Serta penggunaan flexdashboard pada kode di cek kembali agar tidak muncul output text nya seperti tibble atau dataframe. jangan melakukan pemanggilan object lagi. Pada flexdashboard yang ditaruh pada kode hanya langsung kode nya saja (untuk wranglingnya) bisa anda cek lagi pada inclass flexdashboard kelas.  Jika masih terkendala, silahkan melakukan email di mentor@algorit.ma dan melampirkan file zip project anda. 
Terimakasih, semoga membantu

**Reply by manuelatito** [reply]
Dear kak @ramdhinafinita,

Terima kasih kak Fini, saya sudah melakukan email di mentor@algorit.ma dan sudah dijawab oleh team algoritma.
